website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -2441,5 +2466,8 @@ var Range = ace.require('ace/range').Range;
         chunk.skipLines(2, 1, true);
     }
 
-
 })();
+$(function(){
+  $("wmd-help-button").attr("data-toggle","modal");

```

Comment:
Umm, I guess so. I don't actually remember how it makes the box appear though. I'll delete it then.